src/main/resources/db/migration/V43_47__Create_community_zone_table.sql

```diff
@@ -0,0 +1,7 @@
+create table community_zone
```

Comment:
I don't really understand why do you need it as table ?

Because if I understand what you plan to do, a same authorization is set to be used by different zones of different community or same community but different zone ? The doubt for me here is that, why not just adding community_zone_name inside the community_authorization ? 

There is no wrong or true answer but it's about the use case

Reply:
Yes, it's the same community for many zones. Since one community can perform fulldetection multiple times (but checking the used surface limit), I think the community has many authorized zones (to avoid performing detection on the same authorized zone).